src/appmixer/intelswift/auth.js

```
@@ -0,0 +1,42 @@
+const { makeRequest } = require('./commons');
+
+module.exports = {
+
+    type: 'apiKey',
+
+    definition: {
+
+        accountNameFromProfileInfo: 'displayName',
+        
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)